collectors/gsuite/gsuite_collector.js

```diff
@@ -12,45 +12,35 @@
 const moment = require('moment');
 const request = require('request');
 const parse = require('@alertlogic/al-collector-js').Parse;
-// const ManagementClient = require('auth0').ManagementClient;
 const { google } = require('googleapis');
 const { auth } = require('google-auth-library');
 const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
 
-
 const typeIdPaths = [
-    { path: ['etag'] }
+    { path: ['kind'] }
 ];
 
 const tsPaths = [
-    { path: ['id.time'] }
+    { path: ['id','time'] }
 ];
 
-function  listLoginEvents(auth, callback) {
+function  listLoginEvents(auth, params, callback) {
     const service = google.admin({version: 'reports_v1', auth});
-    service.activities.list({
-        userKey: 'all',
-        applicationName: 'login',
-        maxResults: 10,
-    }, (err, res) => {
-        // console.log('SERVICE.ACTIVITES.LIST');
-        // console.log(res);
-        // console.log('----------------------');
-        // console.log("error : ",err);
-        if (err) return callback(null, err);;
+    params['userKey'] = 'all';
+    params['applicationName'] = 'login';
+    service.activities.list(params, (err, res) => {
+        if (!('items' in res.data)) return Error('No more logs');; 
```

Comment:
I'm not sure we should returning an error here, when there are no logs to collect. This isn't exactly an error state, what do you think @JDragovichAlertLogic ?

Reply:
I have checked this, there is no `items` key available if there is no data. So would you rather an empty array be returned?

Reply:
Yes. I think it would be more appropriate than an error.

So there is no `items` key even if the api returns 200 and there is no data?

Reply:
Please the output of the google sdk below. 
In the data section, you will find that the key is missing and the status is 200.

```json
{
  "config": {
    "url": "https://www.googleapis.com/admin/reports/v1/activity/users/all/applications/login?startTime=2019-12-19T17%3A10%3A32.680Z",
    "method": "GET",
    "paramsSerializer": [Function],
    "headers": {
      "Accept-Encoding": "gzip",
      "User-Agent": "google-api-nodejs-client/0.7.2 (gzip)",
      "Authorization": "Bearer ya29.IoIBtQe5CgVlpKhln7OvkAHnZqxnso92ZEMxyNG7sjTA-72FSJqjqSBOKNzsMgb6mbAKHmiDHVLfWIRz81YW6LWVsnItUKyv-wHmpD5uEirJ_hNFEzVmChhOltlUp9X5RBxtuwrfCBjBfwxMvcXLPpAtiYmRWSp1ZTqYkzdFB_R3i2InZw",
      "Accept": "application/json"
    },
    "params": [Object: null prototype] { startTime: "2019-12-19T17:10:32.680Z" },
    "validateStatus": [Function],
    "responseType": "json"
  },
  "data": {
    "kind": "admin#reports#activities",
    "etag": ""R-y6Xh7vAxkAIl4S8_cfLeIZddE/uWzx3kSEEByRJSAtl70HdDZoxjA""
  },
  "headers": {
    "alt-svc": "quic=":443"; ma=2592000; v="46,43",h3-Q050=":443"; ma=2592000,h3-Q049=":443"; ma=2592000,h3-Q048=":443"; ma=2592000,h3-Q046=":443"; ma=2592000,h3-Q043=":443"; ma=2592000",
    "cache-control": "private, max-age=0, must-revalidate, no-transform",
    "connection": "close",
    "content-encoding": "gzip",
    "content-type": "application/json; charset=UTF-8",
    "date": "Thu, 19 Dec 2019 17:15:33 GMT",
    "etag": ""R-y6Xh7vAxkAIl4S8_cfLeIZddE/uWzx3kSEEByRJSAtl70HdDZoxjA"",
    "expires": "Thu, 19 Dec 2019 17:15:33 GMT",
    "server": "GSE",
    "transfer-encoding": "chunked",
    "vary": "Origin, X-Origin",
    "x-content-type-options": "nosniff",
    "x-frame-options": "SAMEORIGIN",
    "x-xss-protection": "1; mode=block"
  },
  "status": 200,
  "statusText": "OK"
}
```

Reply:
Right, that seems like a strange behaviour from the API. But if thats how it is, thats how it is.

In that case we should just return an empty array.

Good spot.